hadoop-hdfs-project/hadoop-hdfs-httpfs/src/test/java/org/apache/hadoop/fs/http/client/BaseTestHttpFSWith.java

```diff
@@ -2081,6 +2085,33 @@ private void testGetFileLinkStatus() throws Exception {
     assertTrue(fs.getFileLinkStatus(linkToFile).isSymlink());
   }
 
+  private void testGetStatus() throws Exception {
+    if (isLocalFS()) {
+      // do not test the getStatus for local FS.
+      return;
+    }
+    final Path path = new Path("/foo");
+    FileSystem fs = FileSystem.get(path.toUri(), this.getProxiedFSConf());
+    if(fs instanceof DistributedFileSystem){
+      DistributedFileSystem dfs = (DistributedFileSystem)
+          FileSystem.get(path.toUri(), this.getProxiedFSConf());
```

Comment:
missing space & code ain't formatted properly
```
    if (fs instanceof DistributedFileSystem) {
      DistributedFileSystem dfs =
          (DistributedFileSystem) FileSystem.get(path.toUri(), this.getProxiedFSConf());
      FileSystem httpFs = this.getHttpFSFileSystem();
```

Reply:
Thank you for your valuable suggestion. I sincerely appreciate it and will promptly implement the required adjustments to the code!